qualtran/bloqs/chemistry/contiguous_register.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from functools import cached_property
+from typing import Dict, Optional, Set, Tuple, Union
+
+import sympy
+from attrs import frozen
+from cirq_ft import TComplexity
+
+from qualtran import Bloq, Register, Signature
+from qualtran.bloqs.basic_gates import TGate
+from qualtran.resource_counting import SympySymbolAllocator
+from qualtran.simulation.classical_sim import ClassicalValT
+
+
+@frozen
+class ContiguousRegister(Bloq):
```

Comment:
is this the name used in the referenced work? It's very confusing in this context because it is not -- in fact -- a `Register`. Off hand, I'd suggest something like `ToContiguous(Bloq)`

Reply:
It is yeah. ToContiguous sounds fine to me.